### ISSUE TITLE
asking for the part of the code related to loss

### ISSUE THREAD

**Issue by cppcute-pm** [alexjunholee/LC2_crossmatching#3]
Hello, Alex!
I'm very appretiated of your amazing work!
I'm realy courious about how the loss function and the supervising signal  (degree of similarity) is formed, could you release the training code as soon as possible ?
Best wishes.
